data/json/items/containers/containers.json

```
@@ -2405,6 +2405,70 @@
     ],
     "melee_damage": { "bash": 1 }
   },
+  {
+    "id": "dvd_case",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "DVD case" },
+    "description": "A protective container designed to hold a Digital Versatile Disc (or DVD, for short).  It is made of plastic and, before the Cataclysm, was used to package and store DVDs, providing protection against scratches, dust, and damage.",
+    "weight": "100 g",
+    "volume": "184 ml",
+    "longest_side": "19 cm",
```

```suggestion
    "weight": "70 g",
    "volume": "175 ml",
    "longest_side": "14 cm",
```

Took a few CD cases, pulled out the CD and anything else inside and weighed them.  Measured them and they all seem to be the same size of 12.5cm tall * 14cm front-to-back * 1cm wide

---

Thank you so much for providing me with exact and direct values, gentleman. I really appreciate it.